src/applications/disability-benefits/all-claims/pages/toxicExposure/toxicExposureConditions.js

```diff
@@ -19,7 +22,13 @@ export const uiSchema = {
       updateUiSchema: makeTEConditionsUISchema,
     }),
   },
-  'ui:validations': [validateTEConditions],
+  // View fields for tracking state changes - hidden from user
+  'view:previousToxicExposureConditions': {
```

Comment:
Double checking: these are still necessary because of the review page is that correct? So even with a custom page we have to have some properties in the schema?

Reply:
Yeah pretty sure its required by forms system